Dockerfile

```diff
@@ -18,6 +18,9 @@
 
 FROM adoptopenjdk/maven-openjdk11
 
+RUN apt-get update && apt-get install -y nodejs npm
+RUN npm cache clean -f && npm install -g n && n stable
```

Comment:
What are these two lines for? Where do we need npm/nodejs?

Reply:
I have created a new PR to optimize Docker image size:

https://github.com/google/fhir-access-proxy/pull/8

will merge this PR